### DISCUSSION TITLE
confidence level of segmentation

### DISCUSSION THREAD

**Discussion by DoDucNhan** [open-mmlab/mmsegmentation#2256] [Q&A]
Hello everyone. I use the BEiT network for semantic segmentation and have trouble determining if my model segments the object correctly. In this case, there won't be any ground truth to validate. The result from "inference_segmentor" is only one array with the class id.

Since I try to extract a specific object (e.g. barrel) from a list of images, I want to apply a threshold to remove bad segmented objects. Is there any way I can obtain a confidence level that tells how confident a model is about that segmentation?

**Comment by changwsh12** [comment]
Please refer to
https://github.com/open-mmlab/mmsegmentation/issues/3528#issuecomment-1974978592
